packs/hpe-icsp/actions/icsp_job_status_get.yaml

```diff
@@ -0,0 +1,26 @@
+---
+  name: "icsp_job_status_get"
+  runner_type: "run-python"
+  description: "Return Status of specified Job"
+  enabled: true
+  entry_point: "icsp_job_status_get.py"
+  parameters:
+    job_id:
+      type: "integer"
+      description: "URI for Job, blank will return all jobs listed"
+      required: false
+      position: 0
+    monitor:
+      type: boolean
+      description: "Monitor Progress of The Job. False = One time check"
+      required: false
+      default: false
```

Comment:
Yeah, i'm also fine with keeping the existing name.

With default argument values I mean that every parameter which has a default value in the metadata should have a default value in the Python action file inside the run method signature.